src/responsibleai/docker_env/Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM mcr.microsoft.com/azureml/openmpi4.1.0-ubuntu20.04:20221010.v1
 
 RUN apt-get -y update && apt-get -y install wkhtmltopdf
 
-ENV AZUREML_CONDA_ENVIRONMENT_PATH /azureml-envs/responsibleai-0.24
+ENV AZUREML_CONDA_ENVIRONMENT_PATH /azureml-envs/responsibleai
```

Comment:
perhaps a silly question: what does that version 0.24 refer to? Do we need it? I'm not sure where it's coming from.

Reply:
I think that was from v0.24 when we publish responsibleai and raiwidgets v0.24.0. Since then the environment was always named to responsibleai-0.24. Now using name `responsibleai` without version fixed is same as azureml-assets repo